tests/integrations/django/test_transactions.py

```diff
@@ -95,12 +95,35 @@ def test_resolver_path_multiple_groups():
     django.VERSION < (2, 0),
     reason="Django>=2.0 required for <converter:parameter> patterns",
 )
+@pytest.mark.skipif(
+    django.VERSION > (5, 0),
```

Comment:
Caution: This will skip tests on Django v5.0.6.
```suggestion
    django.VERSION > (5, 1),
```
```
>>> import django
>>> django.VERSION
(5, 0, 6, 'final', 0)
>>> django.VERSION > (5, 0)
True
>>> django.VERSION > (5, 1)
False
```

Reply:
Perhaps this is intentional.

Reply:
No, this was an oversight, thanks for catching this.